pandas/tests/test_resample.py

```diff
@@ -2165,6 +2165,14 @@ def test_resample_datetime_values(self):
         res = df['timestamp'].resample('2D').first()
         tm.assert_series_equal(res, exp)
 
+    def test_resample_apply_with_additional_args(self):
+        def f(data, add_arg):
```

Comment:
A better test would be to return `add_arg`, to ensure that the proper value is passed through.